generators/workspace/templates/plain/config/webpack.dev.js

```diff
@@ -35,8 +35,9 @@ const defaultAppPath = path.join(workspacePath, defaultApp);
 const appPkg = require(path.join(defaultAppPath, 'package.json'));
 const libName = appPkg.name;
 const libDesc = appPkg.description;
-const {entries, registry, vendors, libraryAliases, filesToLoad} = require(path.join(defaultAppPath, '.yo-rc.json'))['generator-phovea'];
+const {entries, registry, vendors, libraryAliases, filesToLoad, copyFiles} = require(path.join(defaultAppPath, '.yo-rc.json'))['generator-phovea'];
 const fileLoaderRegex = filesToLoad && filesToLoad['file-loader'] ? RegExp(String.raw`(.*)\/(${filesToLoad['file-loader']})\.(html|txt)$`) : RegExp(/^$/);
+const copyAppFiles = copyFiles ? copyFiles.map((file) => ({from: base + '/' + defaultApp + '/' + file, to: base + '/bundles' +  file.substr(file.lastIndexOf('/'))})) : [];
```

Comment:
I think the line is outdated and should be updated to:

```suggestion
const copyAppFiles = copyFiles ? copyFiles.map((file) => ({from: path.join(defaultAppPath, file), to: path.join(workspacePath, 'bundles', path.basename(file)) })) : [];
```

I'd also suggest to [use `path.basename()` to get the filename](https://nodejs.org/dist/latest-v14.x/docs/api/path.html#path_path_basename_path_ext).